tutorials/io/saving_games.rst

```diff
@@ -156,8 +153,14 @@ way to pull the data out of the file as well.
             # Call the node's save function.
             var node_data = node.call("save")
 
+            # Creates the helper class to interact with JSON
+            var json = JSON.new()
+
+            # Convert save data into a serialized JSON string
+            var json_string = json.stringify(node_data)
```

Comment:
This can be achieved with `JSON.stringify()` as a static method now. No need to create a new JSON instance.

Reply:
Thanks, I didn't know about that.   
Already updated the PR

Reply:
Something else that I should update?